### ISSUE TITLE
Decide about terminology: GUID or PID

### ISSUE THREAD

**Issue by nickynicolson** [tdwg/apis#69]
Or is this an issue for TDWG TAG?

**Comment by sharifX** [comment]
Why not both? Technically PIDs such as DOIs are GUIDs but generally GUIDs do not have the registration and governance authority behind the identifier management (such as the DOI foundation). 

https://en.wikipedia.org/wiki/Digital_object_identifier
`Other "identifier systems" may be enabling technologies with low barriers to entry, providing an easy to use labeling mechanism that allows anyone to set up a new instance (examples include Persistent Uniform Resource Locator (PURL), URLs, Globally Unique Identifiers (GUIDs), etc.), but may lack some of the functionality of a registry-controlled scheme and will usually lack accompanying metadata in a controlled scheme. The DOI system does not have this approach and should not be compared directly to such identifier schemes. Various applications using such enabling technologies with added features have been devised that meet some of the features offered by the DOI system for specific sectors (e.g., ARK).`

**Comment by ben-norton** [comment]
Proposal: Everything within the system is referenced with a GUID. Resources with external unique identifiers such as DOIs are annotated accordingly.
Let's say you are hosting a checklist such as the Mammal Diversity Database.
Within the API, especially the requests, the MDD is referenced using a GUID. Resource requests use the GUID.
MDD as a dataset has a DOI, which is provided in the responses. You can request the MDD using the DOI, but it is provided in the responses. See: https://data.naturalsciences.org/api/v2/data-products/checklists/mdd?limit=1

**Comment by ianengelbrecht** [comment]
https://en.m.wikipedia.org/wiki/Persistent_identifier

Seems the internet thinks GUIDs and UUIDs are the same thing (but I don’t): https://en.m.wikipedia.org/wiki/Universally_unique_identifier

IMO We need identifiers for things like taxa, people, localities, etc, that can be used to resolve them unambiguously across different systems/databases. PIDs can’t do that, GUIDs can.

**Comment by sharifX** [comment]
I agree GUIDs and UUIDs are not the same. But what you mean by "PIDs can’t do that, GUIDs can.". Can you elaborate? Why can't PID like DOI be used for taxa?

**Comment by ianengelbrecht** [comment]
Thanks Sharif. The question for taxa for me is what would the DOI resolve to? A Catalog of Life record? Encyclopedia of Life? A record in a taxon specific database like World Spider Catalog? For taxa we have many bits of information spread across many different systems. And as a user I want an identifier I can use to make sure I’m retrieving information for the right taxon across those systems. What’s happening is that we end up using the name to resolve the PID for each system, which seems to me like getting it the wrong way round. Of course a DOI can work, if all those different systems agree to use the DOI as their identifier for that taxon. But then your PIDs would look something like https://eol.org/taxon/10.1109/5.770879 (yeesh!!!), and we start down the road of which identifier technology should be used (LSID, UUID, DOI, handles, etc). At least that’s how I understand the current picture.

**Comment by ben-norton** [comment]
We really need a central doi/guid/pid registry.

**Comment by ben-norton** [comment]
When is a GUID not a version 4 UUID that meets the RFC 4122 specification?

Ben Norton
Head of Technology
Collections Data Curator
Collections Manager, Mineralogy
Research & Collections
North Carolina Museum of Natural Sciences
https://naturalsciences.org/staff/ben-norton
***@***.******@***.***>
919.707.9947

[Sent from Mobile Device]

________________________________
From: Ian Engelbrecht ***@***.***>
Sent: Friday, July 1, 2022 5:30 PM
To: tdwg/apis
Cc: Norton, Ben; Comment
Subject: [External] Re: [tdwg/apis] Decide about terminology: GUID or PID (Issue #69)

CAUTION: External email. Do not click links or open attachments unless you verify. Send all suspicious email as an attachment to Report ***@***.***>


https://en.m.wikipedia.org/wiki/Persistent_identifier<https://urldefense.com/v3/__https://en.m.wikipedia.org/wiki/Persistent_identifier__;!!HYmSToo!ZmCAnt8ssUVpWihAtFHq5uGP_Bbb8anZVFw23BDyWDE6-56q38-wvduFri8ehC85g9XZk96qLukLkpGc8mQG3FMtQPOUVow$>

Seems the internet thinks GUIDs and UUIDs are the same thing (but I don’t): https://en.m.wikipedia.org/wiki/Universally_unique_identifier<https://urldefense.com/v3/__https://en.m.wikipedia.org/wiki/Universally_unique_identifier__;!!HYmSToo!ZmCAnt8ssUVpWihAtFHq5uGP_Bbb8anZVFw23BDyWDE6-56q38-wvduFri8ehC85g9XZk96qLukLkpGc8mQG3FMtz0maRhM$>

IMO We need identifiers for things like taxa, people, localities, etc, that can be used to resolve them unambiguously across different systems/databases. PIDs can’t do that, GUIDs can.
